deploy-image/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM gcr.io/google.com/cloudsdktool/cloud-sdk
+
+RUN apt-get update
+
+RUN  apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+
+RUN  curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash
+
+RUN git clone git://github.com/yyuu/pyenv.git .pyenv
```

Comment:
I don't think you need both of these commands?

Reply:
Had to revert this to use both commands as otherwise the build falls over.

Reply:
Hmm... they are essentially doing the same thing. Can you try deleting the first command to see if it works with just:

```
RUN git clone git://github.com/yyuu/pyenv.git .pyenv
```